internal/server/http/handler/v1/user/handler.go

```diff
@@ -187,9 +188,41 @@ func parseURL(rawURL string) (*url.URL, error) {
 // @Param			limit	query		string	false	"Limit"
 // @Router			/v1/user [get]
 // @Security		Keycloak
-func GetAllUsers(_ service.AuthService) fiber.Handler {
+func GetAllUsers(svc service.AuthService) fiber.Handler {
 	return func(c *fiber.Ctx) error {
-		return c.SendStatus(fiber.StatusNotImplemented)
+		page := c.Query("page")
```

Comment:
Are you currently using page and limit somewhere? We need those parameters if we want to use pagination, what we are not doing (yet).

Reply:
Actually, I wanted to ask the group if we need it in the handler. But I will delete it for now since we don't use it 👍